tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2317,4 +2317,121 @@ public function userGetsThePersonalDriveInformationOfUserUsingGraphApi(string $b
 			)
 		);
 	}
+
+	/**
+	 * @When /^the user "([^"]*)" unassigns the role of user "([^"]*)" using the Graph API$/
+	 * @When /^the user "([^"]*)" tries to unassign the role of user "([^"]*)" using the Graph API$/
```

Comment:
how about if we pass the user role in step and retrieve the user-role id from 
```
private $appEntity = [];
```
this help us to avoid making api request for getting user-role id
```
		$response = GraphHelper::getAssignedRole(
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getStepLineRef(),
			$this->featureContext->getAdminUsername(),
			$this->featureContext->getAdminPassword(),
			$userId
		);
		$appRoleAssignmentId = $this->featureContext->getJsonDecodedResponse($response)["value"][0]["id"];
```